src/legacy/core_plugins/console/public/src/components/help_show_panel.tsx

```diff
@@ -17,26 +17,31 @@
  * under the License.
  */
 
-require('./sense_help_example');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { I18nContext } from 'ui/i18n';
+import { OpenHelpPanel } from './help_panel';
 
-import { i18n } from '@kbn/i18n';
-import template from './welcome.html';
+let isOpen = false;
 
-const storage = require('../storage');
+export function helpShowPanel() {
```

Comment:
By the same token, I'd find `showHelpPanel` easier to grok in this case because it's a function.